packages/arb-avm-cpp/data_storage/include/data_storage/aggregator.hpp

```diff
@@ -43,6 +45,8 @@ class AggregatorStore {
     void saveBlockHash(const uint256_t& block_hash, uint64_t block_height);
 
     void reorg(uint64_t block_height);
+    ValueResult<uint256_t> logsProcessedCount() const;
+    rocksdb::Status updateLogsProcessedCount(uint256_t& count);
```

Comment:
Seems like this should have a const ref argument instead of a mutable ref

Reply:
done